pkg/commands/plan/plan.go

```diff
@@ -322,6 +326,40 @@ func (c *PlanCommand) terraformPlan(ctx context.Context) (*RunResult, error) {
 	stdout.Reset()
 	stderr.Reset()
 
+	util.Headerf(c.Stdout(), "Saving output to JSON file")
```

Comment:
nit: might be confusing between GCS and local for "saving"...

```suggestion
	util.Headerf(c.Stdout(), "Writing Local Plan to JSON File")
```